src/article/dto/create-article.dto.ts

```diff
@@ -1,6 +1,12 @@
+import { ApiProperty } from "@nestjs/swagger";
```

Comment:
please use single quotes to be consistent

Reply:
Sure, that's just how my IDE autocomplete works, should look into how to change this to avoid manual fixing each time.

Reply:
Fixed